src/arch/xtensa/configs/override/jasperlake_chrome_waves.config

```diff
@@ -0,0 +1,14 @@
+CONFIG_CAVS_LPS=y
+CONFIG_COMP_CODEC_ADAPTER=y
```

Comment:
@oleh-titov the link you provided it's private and needs a Google address to access it. So, it's largely not relevant for community. You can briefly describe the issues together with maybe an ticket number for your own reference, but pasting links that are not accessible doesn't help much.

Reply:
Thanks @dbaluta, updated